### DISCUSSION TITLE
Multiple frames selection, editing and brushes resources

### DISCUSSION THREAD

**Discussion by andrearot** [keiwando/toonsquid#117] [Q&A]
Hi everyone, my first post here and firstly i wanna say congratulation to the author of Toonsquid, i think this is the BEST animation app on ipad i’ve ever tried!

I have few questions :-)

1 - Is there a way to select **multiple** frames and paste them in the timeline?
2 - Is there a panel dedicated to editing such as color replace, brightness, curves etc? I can’t find it!
3 - Are there any brushes collections or free downloadable ones available? Is it possible to import Procreate brushes?

Many thanks and great work, keep it up!

**Comment by keiwando** [comment]
Thank you very much!

1. If you double tap and drag in the timeline, you can select multiple drawings at once. If all selected drawings are in the same animation layer, then you can also copy and paste them in the same way that you can copy a single selected drawing.
2. There are currently not color adjustment or effects options available but I am planning on adding that in the future.
3. You can import Photoshop brushes, however, ToonSquid does not yet have a "dual brush" feature so Photoshop brushes that use that feature will look slightly different when imported into ToonSquid. (Only their main brush textures will be used by the brush)
https://toonsquid.com/handbook/brushes/brush_import/#abr-compatibility

**Reply by andrearot** [reply]
Hey thank you very much for the answer and congratulations again for this amazing app!

About point number 1, following your directions I still can't select multiple frames. Can I ask you please of uploading a demonstration video or explaining it to me in any other way? I do exactly what you say but unfortunately it doesn't work, I can only select one frame.

Regarding brushes, do you think you will ever implement Procreate brush import? I find many similarities with this app and I say it in a totally positive way.

Regarding the color adjustments/effects I can't wait for them to be included in the App!

Cheers,
Andrea

**Reply by UndeadLamb** [reply]
The way I select multiple frames is double tapping and holding then drag over the other frames that I want to select

**Reply by andrearot** [reply]
Thank you mate, i got it! I wasn't able to multiple selection before i deactivate magnet icon at the bottom... Quite strange but now it works fine :-)
